HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
+{
+    public string Id { get; set; }
+    public string UserName { get; set; }
```

Comment:
I think we should put an upper limit for the length.

Also specify that the column does not accept unicode characters. EFCore by default assumes strings support unicode characters. But we don't want that in this use case.